content/03.heterogeneity.md

```diff
@@ -19,9 +23,9 @@ If non-biological heterogeneity is detectable, common approaches like  reprocess
 
 Dimensionality reduction is, in fact, a type of representation learning (or feature learning), a process of learning low-dimensional representations or composites of features from raw data. 
 Each learned composite feature becomes a new variable - representing a combination of original features - thereby reducing the dimension of the dataset. 
-Representation learning through matrix factorization can extract composite features from transcriptomics datasets made of combinations of gene expression levels found in the training data that are descriptive in some way [@doi:10.1038/s41467-020-14666-6], and use them to interpret test input data [@doi:10.1093/bioinformatics/btq503; @doi:10.1186/s13059-020-02021-3]. 
-Putting constraints on the features learned by the model, through regularization, can help ensure that the learned representations are biologically relevant [TODO: REF]. 
-Low-dimensional representations trained on a collection of transcriptomic data can also be used as input to supervised machine learning methods[@doi:10.1186/s12859-020-3427-8; @doi:10.1016/j.procs.2016.07.014; @doi:10.1098/rsif.2017.0387]
+Representation learning through matrix factorization can extract composite features from transcriptomics datasets made of combinations of gene expression levels found in the training data that are descriptive in some way [@doi:10.1038/s41467-020-14666-6], and use them to interpret test input data.[@doi:10.1093/bioinformatics/btq503; @doi:10.1186/s13059-020-02021-3] 
+Putting constraints on the features learned by the model, through regularization, can help ensure that the learned representations are generalizable and avoid overfitting of the resulting model [@doi:10.1371/journal.pgen.1004754, @doi:10.1002/sim.6782]. 
```

Comment:
Seems reasonable to me! (This _is_ TODO 3, correct?)

Reply:
Yep, correct!